### DISCUSSION TITLE
Celebrity face strength/weight

### DISCUSSION THREAD

**Discussion by juliux32** [lllyasviel/Fooocus#1729] [Q&A]
Hello,

On the internet, I saw that you can make your person look like a specific celebrity or just a little bit like it, but I am not having luck with it. 

For example, I used simple prompt **"Woman in park facing camera"** and then tried to add a celebrity like this **"Woman in park facing camera. Alexandra Daddario:1"** and got this image which looked like **Alexandra Daddario**: 
![2024-01-03_11-29-07_1296](https://github.com/lllyasviel/Fooocus/assets/21074960/1c8da4ec-b254-4828-a402-10dcded32278)

But then I tried to reduce similarity, and no matter what number I put, it generates basically the same image... Here's an example where I put it to **0.01**: 
![2024-01-03_11-27-27_2635](https://github.com/lllyasviel/Fooocus/assets/21074960/5f9dddd9-4423-437c-9ffa-6badd539e3df)

As you can see, the person is identical no matter if it is set to 1 or 0.01... Am I missing something or doing something wrong? Or maybe Fooocus doesn't have the option to lower similarity, and you can only have a celebrity at 100% or don't have a celebrity at all?

As always, any advice, help, or teaching would be appreciated

**Comment by poor7** [comment]
Default settings.
Promt:"`Woman in park facing camera, (Alexandra Daddario:1)`"
Seed:1234
![image1](https://github.com/lllyasviel/Fooocus/assets/154618384/5c3555f0-6158-4471-88f4-2c535aac5857)
Promt:"`Woman in park facing camera, (Alexandra Daddario:-1)`"
Seed:1234
![image2](https://github.com/lllyasviel/Fooocus/assets/154618384/9087eceb-f17a-4960-9c35-39245ad4a0b8)

**Reply by juliux32** [reply]
And what does it mean? Does it mean that I should use a comma instead of a period? Or that celebrity should be written in brackets? Or that positive number means that person will be a celebrity, and a negative number means that it will not?

Also, what kind of "default" Fooocus settings you are using, because mine default Fooocus settings for **Woman in park facing camera, (Alexandra Daddario:-1)** with a seed of **1234** looks like this: 
![2024-01-03_13-15-58_2485](https://github.com/lllyasviel/Fooocus/assets/21074960/97992d18-38f7-443c-b8c2-0dc82cbb3827)

Don't want to be rude, but your message is not very helpful if you just drop it in without any explanation 🙁

**Comment by poor7** [comment]
Here are the "default" settings for version v2.1.860:
```
{
"Prompt": "Woman in park facing camera, (Alexandra Daddario:1)",
"Negative Prompt": "",
"Fooocus V2 Expansion": "Woman in park facing camera, (Alexandra Daddario:1), beautiful, dynamic dramatic bright futuristic, intricate, elegant, highly detailed, extremely color balanced, deep aesthetic, cinematic light, rich colors, inspired, very inspirational, sincere, inspiring, generous, thought, pretty, vibrant, attractive, innocent, epic, stunning, creative, positive, cute, perfect",
"Styles": "['Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp']",
"Performance": "Speed",
"Resolution": "(1152, 896)",
"Sharpness": 2,
"Guidance Scale": 4,
"ADM Guidance": "(1.5, 0.8, 0.3)",
"Base Model": "juggernautXL_version6Rundiffusion.safetensors",
"Refiner Model": "None",
"Refiner Switch": 0.5,
"Sampler": "dpmpp_2m_sde_gpu",
"Scheduler": "karras",
"Seed": 1234,
"LoRA 1": "sd_xl_offset_example-lora_1.0.safetensors : 0.1",
"Version": "v2.1.860"
}
```
![snap 1](https://github.com/lllyasviel/Fooocus/assets/154618384/6130b4c6-e9ff-434e-b8b0-551a82844798)
```
{
"Prompt": "Woman in park facing camera, (Alexandra Daddario:-1)",
"Negative Prompt": "",
"Fooocus V2 Expansion": "Woman in park facing camera, (Alexandra Daddario:-1), beautiful, dynamic dramatic bright futuristic, intricate, highly detailed, expressive, theatrical, delicate, sharp focus, cinematic, fine detail, professional, winning, elegant, sublime, surreal, vibrant colors, epic composition, magical atmosphere, iconic, cute, pretty, inspiring, extremely noble, thought set, fancy",
"Styles": "['Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp']",
"Performance": "Speed",
"Resolution": "(1152, 896)",
"Sharpness": 2,
"Guidance Scale": 4,
"ADM Guidance": "(1.5, 0.8, 0.3)",
"Base Model": "juggernautXL_version6Rundiffusion.safetensors",
"Refiner Model": "None",
"Refiner Switch": 0.5,
"Sampler": "dpmpp_2m_sde_gpu",
"Scheduler": "karras",
"Seed": 1234,
"LoRA 1": "sd_xl_offset_example-lora_1.0.safetensors : 0.1",
"Version": "v2.1.860"
}
```
![snap 2](https://github.com/lllyasviel/Fooocus/assets/154618384/b66cf3e0-798b-453c-97ab-2f4d783e8907)

You can copy and paste them into the input field, after which the "Generate" button will change to "Load Parameters"

**Reply by juliux32** [reply]
I used text checker to find difference between my "default" and your "default" and for some reason, **Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp** added different prompts for us. **Shouldn't these be always the same for everyone?**
![2024-01-03 14-09-29](https://github.com/lllyasviel/Fooocus/assets/21074960/b9178103-f475-4a9c-af77-08bd8c6caeb9)

**Again... Don't want to be rude, but saying "you can copy and paste", does not explain how this celebrity thing is working on Fooocus and how can I properly adjust it** 🙁

**Reply by poor7** [reply]
Bug!?

**Reply by usigg3** [reply]
My experience is that at least the Fooocus V2 adds a GPT2 generated add on to the prompt and since its generated each time it will be different.

**Reply by juliux32** [reply]
@lllyasviel Bug?

**Comment by poor7** [comment]
Does it work fine for you without filters (Style)? This works for me in Colab both with and without filters (Style).
`{
"Prompt": "Woman in park facing camera, (Alexandra Daddario:1)",
"Negative Prompt": "",
"Fooocus V2 Expansion": "",
"Styles": "[]",
"Performance": "Speed",
"Resolution": "(1152, 896)",
"Sharpness": 2,
"Guidance Scale": 4,
"ADM Guidance": "(1.5, 0.8, 0.3)",
"Base Model": "juggernautXL_version6Rundiffusion.safetensors",
"Refiner Model": "None",
"Refiner Switch": 0.5,
"Sampler": "dpmpp_2m_sde_gpu",
"Scheduler": "karras",
"Seed": 1234,
"LoRA 1": "sd_xl_offset_example-lora_1.0.safetensors : 0.1",
"Version": "v2.1.860"
}`
![snap 3](https://github.com/lllyasviel/Fooocus/assets/154618384/77126cef-c667-4c54-a49a-0e5305c54167)
`{
"Prompt": "Woman in park facing camera, (Alexandra Daddario:-1)",
"Negative Prompt": "",
"Fooocus V2 Expansion": "",
"Styles": "[]",
"Performance": "Speed",
"Resolution": "(1152, 896)",
"Sharpness": 2,
"Guidance Scale": 4,
"ADM Guidance": "(1.5, 0.8, 0.3)",
"Base Model": "juggernautXL_version6Rundiffusion.safetensors",
"Refiner Model": "None",
"Refiner Switch": 0.5,
"Sampler": "dpmpp_2m_sde_gpu",
"Scheduler": "karras",
"Seed": 1234,
"LoRA 1": "sd_xl_offset_example-lora_1.0.safetensors : 0.1",
"Version": "v2.1.860"
}`
![snap 4](https://github.com/lllyasviel/Fooocus/assets/154618384/589c506d-62c3-4956-a849-48fd51926ae1)

**Comment by ItsCrea** [comment]
I have repeatedly encountered the problem that when reducing the weight of the token, the image practically does not change. This applies not only to faces.

**Reply by juliux32** [reply]
These things where you put a number at the end of the prompt, for example **(big nose:1.5)** never worked for me. I don't know what I am doing wrong

**Comment by poor7** [comment]
I draw big noses, but the question is how big a nose do you want?
![image](https://github.com/lllyasviel/Fooocus/assets/154618384/909b2f93-1dde-41ab-9d03-0ed61573147d)
```
{
"Prompt": "(exaggerated big nose:1.5), Georgian woman in park facing camera, photo",
"Negative Prompt": "",
"Fooocus V2 Expansion": "",
"Styles": "['Photo Alien', 'Caricature']",
"Performance": "Speed",
"Resolution": "(1152, 896)",
"Sharpness": 2,
"Guidance Scale": 4,
"ADM Guidance": "(1.5, 0.8, 0.3)",
"Base Model": "juggernautXL_version6Rundiffusion.safetensors",
"Refiner Model": "None",
"Refiner Switch": 0.5,
"Sampler": "dpmpp_2m_sde_gpu",
"Scheduler": "karras",
"Seed": 1234,
"LoRA 1": "sd_xl_offset_example-lora_1.0.safetensors : 0.1",
"Version": "v2.1.860"
}
```

**Reply by juliux32** [reply]
This is cartoon/caricature... do the same for a real person 🙂

**Reply by lslsl3q** [reply]
haha
